website_sale_product_minimal_price/static/src/js/website_sale_product_minimal_price.js

```diff
@@ -17,45 +17,45 @@ odoo.define("website_sale_product_minimal_price.shop_min_price", function(requir
                 this.render_price(),
             ]);
         },
-        render_price: async function() {
+        render_price: function() {
             const $products = $(".o_wsale_product_grid_wrapper");
             const product_dic = {};
             $products.each(function() {
                 product_dic[this.querySelector("a img").src.split("/")[6]] = this;
             });
             const product_ids = Object.keys(product_dic).map(Number);
-            const products_min_price = await this._rpc({
+            this._rpc({
```

Comment:
Preserve promise chain...
```suggestion
            return this._rpc({
```